src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -127,13 +127,21 @@
             "group size would be approximately 10 reads.")
     public int MAX_GROUP_RATIO = 500;
 
+    @Option(doc = "Barcode SAM tag (ex. BC)", optional=true)
```

Comment:
Having read group information, in particular the sample name and library identifier are very helpful for debugging QC issues such as sample swaps and library preparation errors.  So I would certainly recommend using RG heavily.